src/main/java/com/elmakers/mine/bukkit/action/builtin/CustomProjectileAction.java

```diff
@@ -568,10 +579,60 @@ else if (reorient)
         Location targetLocation;
         Targeting.TargetingResult targetingResult = Targeting.TargetingResult.MISS;
         Target target = null;
+        
         if (!ignoreTargeting) {
             targeting.start(projectileLocation);
             target = targeting.target(actionContext, distanceTravelledThisTick);
             targetingResult = targeting.getResult();
```

Comment:
Ok, I've merged this and I'm kind of reviewing it more closely now- I don't understand what this whole part is doing, could you please explain?

It seems like the part below this at line 713 should almost be sufficient- just check for minimum distance travelled requirement on hit, and if not met, return a miss- right?

This part is going to be computationally really expensive- all of the distance() calls can be optimized, but retargeting multiple times per tick incurs block checks as well as entity queries, I'm not sure we want it to work this way. What is this doing that isn't covered by the normal per-tick targeting?